js_hw2/js9.js

```diff
@@ -0,0 +1,16 @@
+"use strict"
+
+let sumOfRow3 = 0;
+let valueOfRow3;
+let numberOfRow3=2;
+
+while ( true ) { 
```

Comment:
it is wrong using of while loop. Instead of true try to use condition  1/numberOfRow < 0.001